sphinx/source/docs/user_guide/tools.rst

```diff
@@ -14,6 +14,7 @@ interactions:
 
 * :ref:`userguide_tools_actions`
 * :ref:`userguide_tools_inspectors`
+* :ref:`userguide_tools_edit`
```

Comment:
Text above this says 'There are three categories of tool interactions:' which is no longer consistent with list below. I think this intro could be improves a bit with some fairly minor changes, so I will just push an update the the branch  in a little while.

Reply:
Thanks, should have looked more closely.